src/selection_classic.c

```diff
@@ -31,7 +31,6 @@ CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
     Part of the code comes from the scrot.c file and maintains its authorship.
 */
 
-#include <assert.h>
```

Comment:
Leaving a standard header included doesn't really hurt. `assert.h` isn't even that big of header file, compared to `stdlib` or `unistd` - which might measurably slow compile time down.

Reply:
It's unused. We're having bugs in code without which scrot would still behave the same.